capstone/home.html

```diff
@@ -0,0 +1,134 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="css/style.css" />
+  <link href="https://fonts.googleapis.com/css2?family=Lato&display=swap" rel="stylesheet">
+  <title>Rock & Roll Hall of Fame</title>
+</head>
+<body>
+  <header>
+    <div class="topBar">
+      <i class="fa-brands fa-facebook-f"></i>
+      <i class="fa-brands fa-twitter"></i>
+      <a href="#">English</a>
+      <a href="#">My page</a>
+      <a href="#">Logout</a>
+    </div>
+    <nav>
+    <div class="navbar">
+      <a class="logo" href="home.html" ><img src="images/logo.svg"></a>
+      <div class="nav-items">
+        <a class="aboutPage" href="about.html">About</a>
+        <a href="home.html#program">Program</a>
+        <a href="home.html#join">Join</a>
+        <a href="#sponsor">Sponsor</a>
+        <a href="about.html#news">News</a>
+        <a class="rockHall" href="#rockhall">Rock Hall</a>
+      </div>
+    </div>
+    </nav>
+    <div class="menu">
+      <a class="menu-button open" href="#"><i class="fa-solid fa-bars"></i></a>
+      <a class="menu-button close" href="#"><i class="fa-solid fa-xmark"></i></a>
+      <div hidden class="menu-content">
+        <a class="logo" href="home.html" ><img src="images/logo.svg"></a>
+        <div class="menu-items">
+          <a class="aboutPage" href="about.html">About</a>
+          <a href="home.html#program">Program</a>
+          <a href="home.html#join">Join</a>
+          <a href="#sponsor">Sponsor</a>
+          <a href="about.html#news">News</a>
+        </div>
+      </div>
+    </div>
+  </header>
+  <main>
+    <div class="main">
+      <div class="homeContent home">
+        <h2 class="greet">"Hello! Rock & Roll Fans"</h2>
+        <h1 class="title">ROCK & ROLL HALL OF FAME 2023</h1>
```

Comment:
- [ ] Kindly try to use the section tag for the sections of the page, so the home page contains almost three sections ,so please try to wrap the elements inside the section that's for the semantic reason 😊